### DISCUSSION TITLE
Can Shionna be used with 6G

### DISCUSSION THREAD

**Discussion by alialsahlany** [NVlabs/sionna#374] [Q&A]
Can Shionna be used to predict the path loss model of sixth-generation 6G operating at terahertz frequency?

**Comment by jhoydis** [comment]
Hi,

If you are referring to Sionna RT, then the answers is yes. However, you would need to make sure that you use only materials for which the constituent parameters are known at these frequencies. The stochastic 3GPP 38.901 models are only defined until 100GHz.
